package.json

```diff
@@ -107,6 +110,9 @@
     "lodash.isempty": "^4.4.0",
     "lodash.trimend": "^4.5.1"
   },
+  "resolutions": {
+    "grunt-webpack/deep-for-each": "https://github.com/spalger/js-deep-for-each/releases/download/v2.0.2-fix-missing-lodash-dep/deep-for-each-2.0.2.tgz"
+  },
```

Comment:
What's our story on this dependency, is this something we can contribute a fix for?

Reply:
Aye, https://github.com/moxystudio/js-deep-for-each/pull/23

Once it is released, and assuming that `grunt-webpack` picks up the new version, we can remove this.